packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2440,6 +2441,70 @@ module.exports = function (log, config, oauthdb) {
     });
   };
 
+  cadReminders.keys.forEach((key, index) => {
+    // Template names are generated in the form `cadReminderFirstEmail`,
+    // where `First` is the key derived from config, with an initial capital letter.
+    const template = `cadReminder${key[0].toUpperCase()}${key.substr(1)}`;
+
+    const isFirstCadReminderEmail = index < cadReminders.keys.length - 1;
+
+    const subject = isFirstCadReminderEmail
+      ? gettext('Your Friendly Reminder: How-To Complete Your Sync Setup')
```

Comment:
Small nit: "How-To" is typically used as an adjective ("This is a how-to guide"), whereas this context uses it as a conjunction, without the hyphen ("Here's how to set this up").

Reply:
@jodyheavener I'll let Alex know, thanks for catching it!